curses-client.py

```diff
@@ -272,15 +282,26 @@ def update_messages_window():
 
     messages_win.box()
     messages_win.refresh()
-
-
+    update_packetlog_win()
+
+def update_packetlog_win():
+    if display_log:
+        packetlog_win.clear()
+        packetlog_win.box()
+        # Get the dimensions of the packet log window
+        height, width = packetlog_win.getmaxyx()
+        for i, packet in enumerate(reversed(packet_buffer)):
+            if i < height - 2:
+                logString = f"From: {get_name_from_number(packet['from'])} To: {get_name_from_number(packet['to'])} Port: {packet['decoded']['portnum']} Payload: {packet['decoded']['payload']}"
+                logString = logString[:width - 5] + '-' if len(logString) > width - 5 else logString
```

Comment:
instead of #fffffffffffff, use the channel name

```suggestion
            to_id = convert_to_camel_case(modem_preset_string) if str(packet['to']) == "4294967295" else get_name_from_number(packet['to'])
            if i < height - 2:
                logString = f"From: {get_name_from_number(packet['from'])} To: {to_id} Port: {packet['decoded']['portnum']} Payload: {packet['decoded']['payload']}"
```